src/platform/code-editor/code-editor.component.ts

```diff
@@ -501,174 +501,186 @@ export class TdCodeEditorComponent implements OnInit, AfterViewInit, ControlValu
+     * and emit the onEditorInitialized event.  This is only used in the browser version.
+     */
+    private initMonaco(): void {
+        let containerDiv: HTMLDivElement = this._editorContainer.nativeElement;
+        containerDiv.id = this._editorInnerContainer;
+        this._editor = monaco.editor.create(containerDiv, Object.assign({
+            value: this._value,
+            language: this.language,
+            theme: this._theme,
+            automaticLayout: this._automaticLayout,
+        }, this.editorOptions));
+        setTimeout(() => {
+            this.onEditorInitialized.emit(undefined);
+        });
+        this._editor.getModel().onDidChangeContent( (e: any) => {
+            this._fromEditor = true;
+            this.writeValue(this._editor.getValue());
+        });
+        // need to manually resize the editor any time the window size
+        // changes. See: https://github.com/Microsoft/monaco-editor/issues/28
+        window.addEventListener('resize', () => {
+            this._editor.layout();
+        });
+        this.addFullScreenModeCommand();
     }
-  }
-
-  addFullScreenModeCommand(): void {
-    const that: TdCodeEditorComponent = this;
-    this._editor.addAction({
-        // An unique identifier of the contributed action.
-        id: 'fullScreen',
-       // A label of the action that will be presented to the user.
-        label: 'Full Screen',
-        // An optional array of keybindings for the action.
-        contextMenuGroupId: 'navigation',
-        keybindings: [
-            monaco.KeyCode.F11,
-        ],
-        contextMenuOrder: 1.5,
-        // Method that will be executed when the action is triggered.
-        // @param editor The editor instance is passed in as a convinience
-        run: function(ed: any): void {
-            that.showFullScreenEditor();
-        },
-    });
-  }
-
-  /**
-   * initMonaco method creates the monaco editor into the @ViewChild('editorContainer')
-   * and emit the onEditorInitialized event.  This is only used in the browser version.
-   */
-  private initMonaco(): void {
-    let containerDiv: HTMLDivElement = this._editorContainer.nativeElement;
-    containerDiv.id = this._editorInnerContainer;
-    this._editor = monaco.editor.create(containerDiv, Object.assign({
-        value: this._value,
-        language: this.language,
-        theme: this._theme,
-        automaticLayout: this._automaticLayout,
-    }, this.editorOptions));
-    setTimeout(() => {
-        this.onEditorInitialized.emit(undefined);
-    });
-    this._editor.getModel().onDidChangeContent( (e: any) => {
-        this._fromEditor = true;
-        this.writeValue(this._editor.getValue());
-    });
-    // need to manually resize the editor any time the window size
-    // changes. See: https://github.com/Microsoft/monaco-editor/issues/28
-    window.addEventListener('resize', () => {
-        this._editor.layout();
-    });
-    this.addFullScreenModeCommand();
-  }
 
 }
```

Comment:
@viccsjain
With your latest commit its kind of hard to review because of all the whitespace changes:
https://github.com/Teradata/covalent-code-editor/pull/38/commits/89b724c16fceb94728ce7f16b260fbe7abd24707

Can you update the PR so it only contains your actual changes?